README.md

```diff
@@ -1,9 +1,60 @@
-Rot13 encoder/decoder
+# rot13 encoder/decoder
 
-See sample usage [here](https://github.com/getlantern/flashlight/blob/8ddc8a9e7571bb702bd6006843a64560cec9be8b/config/config.go#L221)
+Rotates a byte value by thirteen characters.
 
-For a standalone decode/encode, supply the input/output file path as env variables and run:
+## Caveat
+
+The name of this package is something of a misnomer: it does not implement
```

Comment:
Ha, I hadn't realized that our "rot13" wasn't even rot13!

Reply:
Yeah, there's an open PR (#1) pointing it out, and I also got tripped up by it when I was trying to decode something of ours, so I figured it would be good to mention!